app/templates/events/view/tickets/discount-codes.hbs

```diff
@@ -0,0 +1,93 @@
+<div class="ui compact stackable grid">
+  <div class="row">
+    <div class="eight wide column">
+      <h2 class="ui header">
+        {{t 'Discount Codes'}}
+      </h2>
+    </div>
+    <div class="eight wide {{unless device.isMobile 'right aligned'}} column">
+      <button class="ui teal button">
+        <i class="plus icon"></i>
+        {{t 'Create a discount'}}
+      </button>
+    </div>
+  </div>
+  <div class="row">
+    <div class="ui hidden divider"></div>
+    <div class="ui {{if device.isMobile 'vertical'}} basic buttons">
+      <div class="ui button active">
+        {{t 'All'}}
+      </div>
+      <div class="ui button">
+        {{t 'Active'}}
+      </div>
+      <div class="ui button">
+        {{t 'Inactive'}}
+      </div>
+    </div>
+    <table class="ui very basic table">
+      <thead>
+        <tr>
+          <th class="one wide">
+            #
+          </th>
+          <th class="one wide">
+            {{t 'Discount code'}}
+          </th>
+          <th class="one wide">
+            {{t 'Discount code URL'}}
+          </th>
+          <th class="one wide">
+            {{t 'Discount Per Ticket'}}
+          </th>
+          <th class="one wide">
+            {{t 'Validity'}}
+          </th>
+          <th class="one wide">
+            {{t 'Status'}}
+          </th>
+          <th class="one wide">
+            {{t 'Action'}}
+          </th>
+        </tr>
+      </thead>
+      <tbody>
+        {{#each model as |discount index|}}
+          <tr>
+            <td>
+              {{index}}
```

Comment:
Index will start from zero. Please use the increment helper and display the correct number. Please pay more attention to such detail. (This was already commented about in a different PR of some else ... Make it a point to read comments in all PRs.)